src/components/groups/GroupCard.js

```diff
@@ -5,106 +5,107 @@ import styled from "styled-components";
 import { device } from "../../styled/device";
 
 const GroupCard = props => {
-
-	return (
-		<GroupInfoCard>
-			<Link to={`/group/${props.group.id}`}>
-				<CardContainer>
-					<TopDiv>
-						{/* <Image src={props.group.image} fluid /> */}
-						<CardImage src={props.group.image} alt="GroupAvatar" />
-					</TopDiv>
-					<MiddleDiv>
-						{/* <hr /> */}
-						<h4>
-							<b>{props.group.group_name}</b>
-						</h4>
-						{/* <p>{props.group.privacy_setting.toUpperCase()}</p> */}
-					</MiddleDiv>
-					<BottomDiv>
-						{/* <hr /> */}
-						<p>
-							{props.group.members ? props.group.members.length : 0} members
-						</p>
-					</BottomDiv>
-				</CardContainer>
-			</Link>
-		</GroupInfoCard>
-	);
+  return (
+    <GroupInfoCard minWidth={props.minWidth}>
+      <Link to={`/group/${props.group.id}`}>
+        <CardContainer>
+          <TopDiv>
+            {/* <Image src={props.group.image} fluid /> */}
+            <CardImage src={props.group.image} alt="GroupAvatar" />
+          </TopDiv>
+          <MiddleDiv color={props.color}>
+            <h4>
+              <b>{props.group.group_name}</b>
+            </h4>
+            {/* <p>{props.group.privacy_setting.toUpperCase()}</p> */}
+          </MiddleDiv>
+          <BottomDiv>
+            <p>
+              {props.group.members ? props.group.members.length : 0} members
+            </p>
+          </BottomDiv>
+        </CardContainer>
+      </Link>
+    </GroupInfoCard>
+  );
 };
 
 const GroupInfoCard = styled.div`
-	display: flex;
-	flex-direction: column;
-	height: 28vh;
-	margin: 2%;
-	width: 40%;
-	box-shadow: 3px 4px 8px 3px rgba(0, 0, 0, 0.2);
-	transition: 0.3s;
-	border-radius: 5%;
+  display: flex;
+  flex-direction: column;
+  height: 28vh;
+  margin: 1%;
+  width: 40%;
+  min-width: ${props => {
+    return props.minWidth || "none";
+  }};
```

Comment:
That's one of the benefits of styled components not explicitly taught in the curriculum :smile: